release-notes/2023-10/MongoDB_Migration/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: ni_systemlink_enterprise_migrate_mongodb
+description: Creates Jobs in the cluster to migrate data from individual service MongoDB instances to a central instance. See README.md for more details.
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
```

Comment:
I didn't write this...it's generated for you when you ask Helm to create a new Chart. I'll commit this suggestion anyway, though.

Reply:
Ah, feel free to ignore any of my comments that go against standards or practices that I don't know about. I was more thorough on this review than I usually am because trying to migrate the ops handbook content has me in WRITER mode.